RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -494,13 +494,39 @@ class ProjectLoader {
         List<String> errors = []
 
         // Earliest check for valid input and output directories. If they are not accessible or writeable.
-        // Start with the input directory
-        errors += checkDirForReadabilityAndExecutability(analysis.getInputBaseDirectory(), "input")
-        errors += checkDirForReadabilityAndExecutability(analysis.getOutputBaseDirectory(), "output")
 
-        // Out dir needs to be writable
-        if (!FileSystemAccessProvider.instance.isWritable(analysis.getOutputBaseDirectory()))
-            errors << (String) "The output was not writeable at path ${analysis.getOutputBaseDirectory()}."
+        // Check if the configuration values are both set, print out warning if one was set empty.
```

Comment:
Comments are not necessary. Remove. If you think the code does not tell this simple story, rather improve the code.